server/management/commands/extract-module-messages.py

```diff
@@ -0,0 +1,43 @@
+import logging
+import os.path
+import pathlib
+import shutil
+import tempfile
+import time
+from django.core.management.base import BaseCommand
+from cjwstate.modules.module_loader import ModuleFiles, ModuleSpec
+from cjwstate.modules.message_catalogs import update_module_catalogs
+import cjwstate.modules
+
+
+logger = logging.getLogger(__name__)
+
+
+def main(directory):
+    cjwstate.modules.init_module_system()
+
+    logger.info(f"Extracting...")
+
+    with tempfile.TemporaryDirectory() as tmpdir:
```

Comment:
This tempfile stuff isn't needed at all. (We use it in `develop-module` because `import_module_from_directory()` copies all files, slowing down development significantly.)